### ISSUE TITLE
Install error in 1.1.21~trusty

### ISSUE THREAD

**Issue by yatesr** [openhie/openinfoman#28]
When doing a fresh install using the 1.1.21 trusty package I get the error below.  Seems the the db is not getting created.  I have successfully upgraded to this version with a previous version installed, which had created the database previously. 

```
+ mkdir -p /var/lib/openinfoman/webapp/dba
+ cp -R /tmp/openinfoman-basex/basex/webapp/dba/common.xqm /tmp/openinfoman-basex/basex/webapp/dba/databases /tmp/openinfoman-basex/basex/webapp/dba/Dockerfile /tmp/openinfoman-basex/basex/webapp/dba/files /tmp/openinfoman-basex/basex/webapp/dba/login.xqm /tmp/openinfoman-basex/basex/webapp/dba/logs /tmp/openinfoman-basex/basex/webapp/dba/modules /tmp/openinfoman-basex/basex/webapp/dba/queries /tmp/openinfoman-basex/basex/webapp/dba/settings /tmp/openinfoman-basex/basex/webapp/dba/static /tmp/openinfoman-basex/basex/webapp/dba/users /var/lib/openinfoman/webapp/dba
+ /var/lib/openinfoman/bin/basex -Vc 'list provider_directory'
/var/lib/openinfoman/.basex: Unknown option 'EVENTPORT'.
/var/lib/openinfoman/.basex: writing new configuration file.
org.basex.core.BaseXException: Database 'provider_directory' was not found.
	at org.basex.core.cmd.Open.open(Open.java:92)
	at org.basex.core.cmd.List.listDB(List.java:121)
	at org.basex.core.cmd.List.run(List.java:51)
	at org.basex.core.Command.run(Command.java:398)
	at org.basex.core.Command.execute(Command.java:100)
	at org.basex.api.client.LocalSession.execute(LocalSession.java:132)
	at org.basex.api.client.Session.execute(Session.java:36)
	at org.basex.core.CLI.execute(CLI.java:103)
	at org.basex.core.CLI.execute(CLI.java:87)
	at org.basex.core.CLI.execute(CLI.java:74)
	at org.basex.BaseX.<init>(BaseX.java:86)
	at org.basex.BaseX.main(BaseX.java:42)
org.basex.core.BaseXException: Database 'provider_directory' was not found.
	at org.basex.core.Command.execute(Command.java:101)
	at org.basex.api.client.LocalSession.execute(LocalSession.java:132)
	at org.basex.api.client.Session.execute(Session.java:36)
	at org.basex.core.CLI.execute(CLI.java:103)
	at org.basex.core.CLI.execute(CLI.java:87)
	at org.basex.core.CLI.execute(CLI.java:74)
	at org.basex.BaseX.<init>(BaseX.java:86)
	at org.basex.BaseX.main(BaseX.java:42)
Database 'provider_directory' was not found.
dpkg: error processing package openinfoman (--configure):
 subprocess installed post-installation script returned error exit status 1
Setting up php5-readline (5.5.9+dfsg-1ubuntu4.14) ...

Creating config file /etc/php5/mods-available/readline.ini with new version
php5_invoke: Enable module readline for cli SAPI
Setting up php5-json (1.3.2-2build1) ...
php5_invoke: Enable module json for cli SAPI
Errors were encountered while processing:
 openinfoman
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by litlfred** [comment]
Looks like a change of command line arguments/env variables from BaseX 8.3.1 to 8.4.1.  I will investigate.

Cheers,
-carl

When doing a fresh install using the 1.1.21 trusty package I get the error below. Seems the the db is not getting created. I have successfully upgraded to this version with a previous version installed, which had created the database previously.

+ mkdir -p /var/lib/openinfoman/webapp/dba
+ cp -R /tmp/openinfoman-basex/basex/webapp/dba/common.xqm /tmp/openinfoman-basex/basex/webapp/dba/databases /tmp/openinfoman-basex/basex/webapp/dba/Dockerfile /tmp/openinfoman-basex/basex/webapp/dba/files /tmp/openinfoman-basex/basex/webapp/dba/login.xqm /tmp/openinfoman-basex/basex/webapp/dba/logs /tmp/openinfoman-basex/basex/webapp/dba/modules /tmp/openinfoman-basex/basex/webapp/dba/queries /tmp/openinfoman-basex/basex/webapp/dba/settings /tmp/openinfoman-basex/basex/webapp/dba/static /tmp/openinfoman-basex/basex/webapp/dba/users /var/lib/openinfoman/webapp/dba
+ /var/lib/openinfoman/bin/basex -Vc 'list provider_directory'
/var/lib/openinfoman/.basex: Unknown option 'EVENTPORT'.
/var/lib/openinfoman/.basex: writing new configuration file.
org.basex.core.BaseXException: Database 'provider_directory' was not found.
    at org.basex.core.cmd.Open.open(Open.java:92)
    at org.basex.core.cmd.List.listDB(List.java:121)
    at org.basex.core.cmd.List.run(List.java:51)
    at org.basex.core.Command.run(Command.java:398)
    at org.basex.core.Command.execute(Command.java:100)
    at org.basex.api.client.LocalSession.execute(LocalSession.java:132)
    at org.basex.api.client.Session.execute(Session.java:36)
    at org.basex.core.CLI.execute(CLI.java:103)
    at org.basex.core.CLI.execute(CLI.java:87)
    at org.basex.core.CLI.execute(CLI.java:74)
    at org.basex.BaseX.<init>(BaseX.java:86)
    at org.basex.BaseX.main(BaseX.java:42)
org.basex.core.BaseXException: Database 'provider_directory' was not found.
    at org.basex.core.Command.execute(Command.java:101)
    at org.basex.api.client.LocalSession.execute(LocalSession.java:132)
    at org.basex.api.client.Session.execute(Session.java:36)
    at org.basex.core.CLI.execute(CLI.java:103)
    at org.basex.core.CLI.execute(CLI.java:87)
    at org.basex.core.CLI.execute(CLI.java:74)
    at org.basex.BaseX.<init>(BaseX.java:86)
    at org.basex.BaseX.main(BaseX.java:42)
Database 'provider_directory' was not found.
dpkg: error processing package openinfoman (--configure):
 subprocess installed post-installation script returned error exit status 1
Setting up php5-readline (5.5.9+dfsg-1ubuntu4.14) ...

Creating config file /etc/php5/mods-available/readline.ini with new version
php5_invoke: Enable module readline for cli SAPI
Setting up php5-json (1.3.2-2build1) ...
php5_invoke: Enable module json for cli SAPI
Errors were encountered while processing:
 openinfoman
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by litlfred** [comment]
I removed the offending EVENTPORT configuration option as it it was removed in version 8.2 of BaseX and we are now running 8.4.

I am rebuilding the packages as right now, and hopefully 1.1.25 will fix the issue.

**Comment by jennifershivers** [comment]
@yatesr and @litlfred Please advise on status

**Comment by yatesr** [comment]
On a fresh ubuntu install I am still getting the same error.

**Comment by litlfred** [comment]
I found the issue and it should be fixed as of the openinfoman-1.1.34 package.  Rebuilding now.
Cheers,
-carl

**Comment by yatesr** [comment]
1.1.34 fixed the issue for me. Thanks!
